payjoin/src/uri.rs

```diff
@@ -24,6 +24,28 @@ impl Payjoin {
             Payjoin::Unsupported => false,
         }
     }
+    pub fn endpoint(&self) -> Option<&Url> {
+        match self {
+            Payjoin::Supported(params) => Some(&params.endpoint),
+            Payjoin::V2Only(params) => Some(&params.endpoint),
+            Payjoin::Unsupported => None,
+        }
+    }
+    pub fn disable_output_substitution(&self) -> bool {
+        match self {
+            Payjoin::Supported(params) => params.disable_output_substitution,
+            Payjoin::V2Only(params) => params.disable_output_substitution,
+            Payjoin::Unsupported => false,
+        }
+    }
```

Comment:
It does not seem like these are used outside of tests and therefore should not be exposed as public methods. These can be gotten direcltly from the `PayjoinUri` struct